audiorecorder/src/main/java/org/odk/collect/audiorecorder/mediarecorder/MediaRecorderRecordingResource.kt

```diff
@@ -66,4 +69,9 @@ internal class AMRRecordingResource(mediaRecorder: MediaRecorder) : MediaRecorde
         mediaRecorder.setAudioSamplingRate(8000)
         mediaRecorder.setAudioEncodingBitRate(12200)
     }
+
+    override fun stop() {
+        resume()
```

Comment:
As far as I can tell from docs and playing around calling `resume()` is safe (even when the recording isn't paused).